### ISSUE TITLE
mAP Evaluation

### ISSUE THREAD

**Issue by LSQI15** [BMW-InnovationLab/BMW-YOLOv4-Training-Automation#9]
Just to clarify - Is the mAP result we get from YOLO API evaluated on the test set or it is calculated on the training set?

" {"success":true,"start_time":"201912261647","data":{"current_training_iteration":"1179","total_loss":"0.750510","average_loss_error":"0.788038","current_learning_rate":"0.001000","total_time":"1.202105","number_of_images":"24759","**mAP":"0.990001"}**}"

Thanks!

**Comment by hadikoub** [comment]
The mAP is calculated for every 4 epochs and will use the validation dataset provided in valid.txt (test set) that is automatically generated based on the train-test ratio you provide in the config file.

**Comment by LSQI15** [comment]
Thanks for the clarification!
